### PR TITLE
count fights against void monsters from the cursed magnifying glass with _voidFights

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

File: src/data/defaults.txt [hunk 1]
```diff
@@ -2068,6 +2068,7 @@ user	_villainLairWebUsed	false
 user	_vitachocCapsulesUsed	0
 user	_vmaskAdv	0
 user	_vmaskBanisherUsed	false
+user _voidFights 0
```
Comment (Rinn) [inline src/data/defaults.txt:2071]: need a tab here not a space

**Review by Rinn — CHANGES_REQUESTED** [review]

File: src/data/defaults.txt (hunk 1 above)
Comment (horrible-little-slime) [reply src/data/defaults.txt]: Thanks!

File: src/net/sourceforge/kolmafia/request/FightRequest.java [hunk 2]
```diff
@@ -2641,6 +2646,10 @@ public static final void updateCombatData(
             }
             break;
 
+          case VOID:
+            Preferences.increment("_voidFights");
```
Comment (gausie) [inline src/net/sourceforge/kolmafia/request/FightRequest.java:2650]: Can you max this out at 5 (there is an overload for doing so)? And can you add a check for the "time seems to stop" message, the failure of which should set this straight to 5.

Comment (horrible-little-slime) [reply src/net/sourceforge/kolmafia/request/FightRequest.java]: aye aye, cap'n

**Review by gausie — APPROVED** [review]
Thank you!

**Comment by gausie** [comment]
Now before I merge @horrible-little-slime would you consider writing a test for FightRequest? There are some in the source to base it off

**Pull request merged**